fv3gfs-physics/fv3gfs/physics/_config.py

```diff
@@ -3,7 +3,7 @@
 
 import f90nml
 
-from pace.util.namelist import NamelistDefaults, namelist_to_flatish_dict
+from pace.util.namelist import Namelist, NamelistDefaults, namelist_to_flatish_dict
```

Comment:
Can these be imported from the package top level, `pace.util`?

Reply:
`Namelist`, `NamelistDefaults` imported from top level now. `namelist_to_flatish_dict` is not used anywhere but inside pace.util.namelist.